apps/system/js/haiku_emoji_panel.js

```diff
@@ -58,6 +58,8 @@
       var target = e.target;
       var receiver = window.app.pairNumber;
       var messageBody = target && target.dataset.icon;
+      var audio = document.getElementsByTagName("audio")[0];
+      audio.play();
```

Comment:
Can you make sure this will play reliably, repeatedly? I've seen issues where an audio snippet will play fine the first time, but glitch subsequent times.